extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -244,8 +219,7 @@ export class AmpSlideScroll extends BaseSlides {
       if (this.scrollTimeout_) {
         Services.timerFor(this.win).cancel(this.scrollTimeout_);
       }
-      const timeout = this.shouldDisableCssSnap_ ? IOS_TOUCH_TIMEOUT
```

Comment:
`shouldDisableCssSnap_` is also set on iOS 10.3 because they had a bug with  CSSSnappoints, needs to continue to exist. (we essentially disabled nativeSnapPoints support for iOS = 10.3 eventhough it does support native snap points )

Reply:
Got it!

Reply:
oh, this PR https://github.com/ampproject/amphtml/pull/17058/files removed the iOS 10.3 hack. carousel might be a bit broken on 10.3 ( the issue with 10.3 wasn't too bad, but at the time when it had 80% market share it was not optimal). Anyway, now it is at 5% and I am okay with this going away.